Scripts/script-ParseCSV.yml

```diff
@@ -7,10 +7,11 @@ script: |
   import socket
 
   def unicode_dict_reader(utf8_data, **kwargs):
-      csv_reader = csv.DictReader(utf8_data, **kwargs)
+      csv_reader = csv.DictReader((line.replace('\0', '') for line in utf8_data), **kwargs)
```

Comment:
Please add a short comment here to describe why this string replacement is required, something like `removing null bytes..`